style.css

```diff
@@ -309,3 +314,164 @@ ul .languages {
 footer .social_media_icons {
   margin-bottom: 19px;
 }
+
+.break_line {
+  display: none;
+}
+
+/* Media queries */
+
+@media screen and (min-width: 769px) {
+  /*  Header Section */
+
+  .header {
+    display: none;
+  }
+
+  .navbar_destop_ver {
+    background-color: transparent;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    position: absolute;
+    top: 50px;
+  }
+
+  .navbar_destop_ver a {
+    text-decoration: none;
+    color: #172b4d;
+    font-weight: 600;
+    margin: auto 40px;
+  }
+
+  /* Headline Section */
+
+  .headline {
+    background-image: url("./images/Header-llustration-desktop.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 1300px;
+    margin-top: 0;
+  }
+
+  .whoiam {
+    line-height: 60px;
+    font-size: 48px;
+    margin-top: -350px;
+  }
+
+  .job {
+    margin-top: 25px;
+  }
+
+  .abt_me {
+    margin-top: -10px;
+    margin-bottom: 250px;
+    padding-left: 355px;
+    padding-right: 355px;
+  }
+
+  section .social_media_icons {
+    margin-top: -200px;
+  }
+
+  /* Works Section */
+
+  hr {
+    display: none;
+  }
+
+  .cards {
```

Comment:
- [ ] You have done a great job with your desktop version design :raised_hands: but there is an introduced issue in your mobile design because the cards of the projects and the about section are aligned to the left. Please, fix this alignment issue in order to match the Figma design :smiley: :pray:

  Cards:
  ![image](https://user-images.githubusercontent.com/81522435/144503616-56c88553-2aa4-445b-b6a6-7049bdd7cd70.png)

  About section:
  ![image](https://user-images.githubusercontent.com/81522435/144503694-20137b70-e7e8-4dcb-a011-f21e62016556.png)

  - I think you could use the same method that you used to align the "Skills cards":
  ![image](https://user-images.githubusercontent.com/81522435/144503863-207f866c-c292-4d62-b809-563c3db75cdc.png)